src/components/FilePreview/components/FileRenderer/BaseRenderers/textHooks.test.js

```diff
@@ -32,15 +29,21 @@ describe('textHooks', () => {
   });
 
   describe('useTextRendererData', () => {
+    let setStateSpy;
+    const setValue = jest.fn();
+
     beforeEach(() => {
+      setStateSpy = jest.spyOn(React, 'useState').mockImplementation((value) => [value, setValue]);
+    });
+
+    afterEach(() => {
+      setStateSpy.mockRestore();
       jest.clearAllMocks();
-      state.mock();
     });
-    afterEach(() => { state.resetVals(); });
 
     it('start with empty content', () => {
       useTextRendererData({});
-      state.expectInitializedWith(stateKeys.content, '');
+      expect(setStateSpy).toHaveBeenCalledWith('');
```

Comment:
It'd be great to add a comment here similar to the ones added in `FileRenderer/hooks.test.js`.